model/src/main/java/hu/blackbelt/judo/meta/jsl/validation/JslDslValidator.xtend

```diff
@@ -1736,7 +1736,16 @@ class JslDslValidator extends AbstractJslDslValidator {
                 JsldslPackage::eINSTANCE.entityMemberDeclaration.getEStructuralFeature("ID"),
                 INVALID_DECLARATION)
     	}
-    	
+
+        if (field.identifier &&
+            field.referenceType instanceof PrimitiveDeclaration &&
+            TypeInfo.getTargetType(field.referenceType).isNumeric &&
+            ((field.referenceType as Modifiable).getModifier(JsldslPackage::eINSTANCE.scaleModifier) as ScaleModifier).value > BigInteger.ZERO ) {
+            error("An identifier field cannot be decimal number.",
```

Comment:
Change message to: Identifier cannot be numeric type with scale

Reply:
+1

Reply:
[c4cd2a4](https://github.com/BlackBeltTechnology/judo-meta-jsl/pull/170/commits/c4cd2a47652917be76d56b6678760cbf72164d63)